tools/legal-review/engine/report-state

```diff
@@ -1,3 +1,3 @@
-1B53BC9C81329A6179E621F07FD2E05E655D0A2BA5B92ED15057676D34EE4737
-E6F042346A7544959A9B053257F92C1885133D5ABC00D736474FF76C5F02F5EC
-0
+40D9E7D8EDB6EE17D0F28F1420B7D6EF425750A0428E020CF99CB56A3F28259C
+E7BF035F18ADC6579B5FAB9B15E3E63A571A3DE99E4F790BD47ACFBA4FAD9B6B
+7
```

Comment:
There are some pending problems in the legal review tool used for gathering dependencies of Open Source dependencies we ship.

Please run `sbt openLegalReviewReport` to see and review them. [These](https://enso.org/docs/developer/enso/distribution/licenses.html) [docs](https://enso.org/docs/developer/enso/distribution/licenses.html#review-process) may be helpful. Let me know if you need any futher help on that.